### PR TITLE
new commit and changing

### PR DESCRIPTION
**Pull request by tchasinga**
General requirements
================================
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
Make sure that you used correct [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
JavaScript requirements
Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
Project requirements
Implement only a basic UI with plain HTML:
Basic UI

Create a collection that keeps a list of books (hint: you can use an array of objects for that).
Create a function to add a new book to the collection, with title and author.
Create a function to remove a book from the collection (hint: you can use the [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method).
Display all books saved in the collection in the top part of the page.
Make sure that when a user clicks the "Add" button:
A new book is added to the collection.
The new book is displayed in the page.
Make sure that when a user clicks the "Remove" button:
The correct book is removed from the collection.
The correct book dissapears from the page.
Make sure that data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md).
To debug your application, you can use [console.log()](https://developer.mozilla.org/en-US/docs/Web/API/Console/log). Remember to remove debugging code before pushing your code to the remote repository.
Don't use any library or framework, use only plain JavaScript.
TIP: If you separated the code in multiple files and you have the following issues with the linter: (i) complaining about having multiple classes at the same file, or (ii) unused variables, disable the linter check for that rule in the file with a /* eslint-disable rule-you-want-to-disable */.

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
Hi @tchasinga and @badger-99,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that does not have a customized readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js/sectionCollector.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/* eslint-disable no-use-before-define */
+// Book class : book presentation
+// eslint-disable-next-line max-classes-per-file
```
Comment (DuaneDave) [inline js/sectionCollector.js]: Consider not disabling the linters, instead resolve the errors you are having to ensure that you are following best practices.

File: js/sectionCollector.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  UI.deleteBook(e.target);
+  Store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+});
+
+// const removeBTN = document.querySelector('.delete');
+
+// removeBTN.addEventListener('click',() => {
+//     // Store.removeBook
+//     console.log('remove the book now')
+// })
```
Comment (DuaneDave) [inline js/sectionCollector.js]: To ensure that you're following best practices, consider removing unused code pieces from your codebase.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @tchasinga and @badger-99 👋, 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good font selection
✔️ Well written code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @tchasinga, and @badger-99 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**